### PR TITLE
SFCC Task - PDP

### PR DESCRIPTION
**Pull request by stanislava-pavlovaa**

### CODE REVIEW

File: cartridges/app_custom_pdp/cartridge/client/default/scss/product/detail.scss
Comment (kalipiev) [inline cartridges/app_custom_pdp/cartridge/client/default/scss/product/detail.scss:1]: Try using utility classes. Examples: padding, margin, border, display, text-align, etc
Font-related properties to come from variables in _variables.scss

File: cartridges/app_custom_pdp/cartridge/controllers/Product.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+"use strict";
+
+var server = require("server");
+server.extend(module.superModule);
+
+server.append("Show", function (req, res, next) {
+    var viewData = res.getViewData();
+
+    var ContentMgr = require("dw/content/ContentMgr");
+    var asset = ContentMgr.getContent("promotional-message");
```
Comment (kalipiev) [inline cartridges/app_custom_pdp/cartridge/controllers/Product.js]: Is this asset called multiple times? If not we can just include it in the template with iscontentasset

File: cartridges/app_custom_pdp/cartridge/templates/default/product/components/recommendationsCarousel.isml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            <h3 class="text-center carousel-title">${Resource.msg('product.recommendations', 'product', null)}</h3>
+        </div>
+    </div>
+
+    <div id="recommendationsCarousel" class="carousel slide w-100" data-ride="carousel">
+        <div class="carousel-inner w-100 d-flex align-self-stretch" role="listbox">
+            <isloop items="${product.recommendations}" var="recommendation" status="loopStatus">
+                <div class="carousel-item ${loopStatus.first ? 'active' : ''}">
+                    <div class="col-3">
+                        <isinclude url="${URLUtils.url('Tile-Show', 'pid', recommendation, 'swatches', false, 'ratings', true, 'showQuickView', false)}" />
```
Comment (kalipiev) [inline cartridges/app_custom_pdp/cartridge/templates/default/product/components/recommendationsCarousel.isml:18]: Like this the product images will render imediately. Please look for an option to lazy load them.

Comment (stanislava-pavlovaa) [reply cartridges/app_custom_pdp/cartridge/templates/default/product/components/recommendationsCarousel.isml:18]: Fixed

**Pull request merged**